src/main/java/org/opensearch/sdk/sample/helloworld/rest/RestHelloAction.java

```diff
@@ -66,4 +125,21 @@ private ExtensionRestResponse handleBadRequest(ExtensionRestRequest request) {
         return new ExtensionRestResponse(request, NOT_FOUND, "Extension REST action improperly configured to handle " + request.toString());
     }
 
+    private String parseJsonAdjective(String json) {
+        // TODO: Once CreateComponents has an XContentRegistry available we can parse from there
```

Comment:
Let's create a follow up issue for this. This will be needed.

Reply:
I kind of figured you were already doing that as part of your AD detector and I'd just steal it here :)

Reply:
#178